docs/assets/js/contact-form.js

```diff
@@ -0,0 +1,44 @@
+var params = window.location.search;
+if (params === "?plan=enterprise") {
+  var plan = document.getElementById("enterprise");
+  plan.checked = true
+} else {
+  var plan = document.getElementById("pro");
+  plan.checked = true
+}
+
+/* Contact form */
+$(function() {
+  var submitButton = $("#submit-button");
+  var form = $("#contact-form");
+
+  function submitForm() {
```

Comment:
This doesn't seem to actually submit the form? I see error checking... but where's the submit?